src/metabase/metabot/client.clj

```diff
@@ -1,15 +1,16 @@
 (ns metabase.metabot.client
   (:require
-   [cheshire.core :as json]
-   [metabase.metabot.settings :as metabot-settings]
-   [metabase.util.log :as log]
-   [wkok.openai-clojure.api :as openai.api]))
+    [cheshire.core :as json]
+    [clojure.core.memoize :as memoize]
+    [metabase.metabot.settings :as metabot-settings]
+    [metabase.util.log :as log]
+    [wkok.openai-clojure.api :as openai.api]))
 
 (set! *warn-on-reflection* true)
 
-(def ^:dynamic *bot-endpoint*
+(def ^:dynamic *create-chat-completion-endpoint*
```

Comment:
usually better to do something like

```clojure
(defn- default-chat-completion-endoint [params options] ...)
(def ^:dynamic *create-chat-completion-endpoint* default-chat-completion-endpoint)
```

bonus points for putting arglist metadata on the dynamic var.

Reply:
Done